ReactWindows/ReactNative.Net46/Bridge/DispatcherHelpers.cs

```diff
@@ -5,8 +5,43 @@
 
 namespace ReactNative.Bridge
 {
-    static class DispatcherHelpers
+    internal static class DispatcherHelpers
     {
+        private static Dispatcher _dispatcher;
+
+        internal static Dispatcher CurrentDispatcher
+        {
+            get
+            {
+                if (_dispatcher == null)
+                {
+                    throw new InvalidOperationException("Dispatcher has not been set");
+                }
+
```

Comment:
Take this, insert into private method called `AssertDispatcherSet`, apply below.